README.md

```diff
@@ -367,7 +367,8 @@ values (EG: "default_server": True)
                                 "share_adjusters": [
                                     {
                                         "share_adjuster_class": "aurproxy.share.adjusters.RampingShareAdjuster",
```

Comment:
That's funny, I actually fixed that in my local config.

Is there any reason for the tellapart namespace or would it make sense to move the app up to just aurproxy...?

I meant to previously ask about that.